components/ProtegeProgramCard.tsx

```diff
@@ -0,0 +1,45 @@
+import {Box, Stack, Typography} from '@mui/joy';
+import Card from '@mui/joy/Card';
+import DarkOutlineButton from './DarkOutlineButton';
+import Image from 'next/image';
+
+export default function ProtegeProgramCard({ icon, text, type, description }: any) {
+  return (
+    <Card
+      variant='outlined'
+      className='border-round'
+      sx={{
+        width: {xs: '300px', lg: '350px'},
+        height: 'auto',
+        border: '2px solid #93AED3',
+        backgroundColor: 'white'
```

Comment:
to add the hover thing like sophia said, maybe you can add something like this: 
        '&:hover':{
          borderColour: '#93AED3',
          boxShadow: '0 4px 8px rgba(0, 0, 0, 0.2)'
        } and experiment to see what looks good!

Reply:
similarly, i think when hovering on the card, if there is a blue overlay on the card, like really light blue overlay. That might also look good. Some thing like this with a transition         transition: 'background-color 0.5s ease',
        '&:hover': {
          backgroundColor: 'rgba(147, 174, 211, 0.1)', 
        },